detox/e2e/init.js

```diff
@@ -1,13 +1,17 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {System, User} from '@support/server_api';
+import {Plugin, System, User} from '@support/server_api';
+
+import testConfig from '../test_config';
 
 beforeAll(async () => {
     // Login as sysadmin and reset server configuration
-    await System.apiCheckSystemHealth();
-    await User.apiAdminLogin();
-    await System.apiUpdateConfig();
+    const baseUrl = testConfig.siteUrl;
```

Comment:
For now, I'm only adding the default url for initial setup. Once we have test cases for multiple servers, I will modify this code so that we have at max 3 different server urls setup.